socorro/tests/external/es/test_connection_context.py

```diff
@@ -2,62 +2,62 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-import datetime
-
+from socorro import settings
+from socorro.libclass import build_instance
 from socorro.lib.libdatetime import utc_now
 
 
 class TestConnectionContext:
-    def test_create_index(self, es_conn):
-        # Delete any existing indices first
-        for index in es_conn.get_indices():
-            es_conn.delete_index(index)
-        es_conn.refresh()
-        es_conn.health_check()
+    def build_conn(self):
+        return build_instance(
+            class_path="socorro.external.es.connection_context.ConnectionContext",
+            kwargs=settings.CRASH_DESTINATIONS["es"]["options"],
+        )
 
-        template = es_conn.get_index_template()
-        now = utc_now()
-        index_name = now.strftime(template)
+    def test_create_index(self, es_helper):
+        # Delete any existing indices first
+        es_helper.delete_indices()
 
-        # Create the index and make sure it's there
-        es_conn.create_index(index_name)
-        es_conn.health_check()
-        assert index_name in list(es_conn.get_indices())
+        es_helper.refresh()
+        es_helper.health_check()
 
-    def test_delete_index(self, es_conn):
-        template = es_conn.get_index_template()
+        template = es_helper.get_index_template()
         now = utc_now()
         index_name = now.strftime(template)
 
-        es_conn.create_index(index_name)
-        es_conn.health_check()
-        assert index_name in list(es_conn.get_indices())
-
-        es_conn.delete_index(index_name)
-        assert index_name not in list(es_conn.get_indices())
-
-    def test_delete_old_indices(self, es_conn):
```

Comment:
"Old indices" relates to the retention policy which is no longer part of the connection context--it was moved to crashstorage code. This test was also moved.